db/migrate/20230711155304_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :author, null: false, foreign_key: true
```

Comment:
- [ ] I am having some issues running the migrations. I have checked the migrations and could not find the migration to create the `comments` table. Here you are trying to add an author reference before creating the comments table resulting in the following error. Please attend to the issue. 🤝️

![image](https://github.com/prius2055/rails-blogging-app/assets/17243325/3dad2c58-0e6e-44dd-a96b-bd99b5b8c199)